python/pyarrow/parquet/core.py

```diff
@@ -761,9 +761,9 @@ def _sanitize_table(table, new_schema, flavor):
     by flavor argument. This take priority over the coerce_timestamps option.
 coerce_timestamps : str, default None
     Cast timestamps to a particular resolution. If omitted, defaults are chosen
-    depending on `version`. By default, for ``version='1.0'`` (the default)
-    and ``version='2.4'``, nanoseconds are cast to microseconds ('us'), while
-    for other `version` values, they are written natively without loss
+    depending on `version`. For ``version='1.0'`` and ``version='2.4'``,
```

Comment:
Should we use ` `` ` for `version` too?

```suggestion
    depending on ``version``. For ``version='1.0'`` and ``version='2.4'``,
```

Reply:
Strictly speaking not, because in that case we are referring to the keyword, which uses single backticks following the numpydoc guidelines (although the difference is a bit ambiguous)